### PR TITLE
add fluentd-gcp repo

### LINKED ISSUE #79432: Move fluentd-elasticsearch out of k/k
**What would you like to be added**:
currently the fluentd-elasticsearch cluster addon creates two docker images that are hosted in non-kubernetes controlled repositories and the code to create them is hosted in k/k.  either the code should be moved out of k/k or the images to should be controlled by the kubernetes org.

**Why is this needed**:
this type of ownership mismatch will lead to a bad user experience and possible security risks as and when the management for these artifacts get out of sync.

### PR DESCRIPTION
**Pull request by coffeepac**
adds fluentd-gcp from https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-gcp
fluentd-gcp will be remove in k/k in: https://github.com/kubernetes/kubernetes/pull/101132
fixes: kubernetes/kubernetes#79432

### CODE REVIEW

File: OWNERS_ALIAS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+aliases:
```
Comment (ehashman) [inline OWNERS_ALIAS]: Remove this file

**Review by ehashman — CHANGES_REQUESTED** [review]

**Comment by ehashman** [comment]
/verify-owners

should work now

**Comment by ehashman** [comment]
/hold

to remove redundant OWNERS_ALIAS file

**Comment by coffeepac** [comment]
@ehashman errant file removed, thank you for sorting that out!

**Review by ehashman — COMMENTED** [review]
/hold cancel
/lgtm
/approve

**Pull request merged**